llo/plugin_outcome.go

```diff
@@ -234,8 +235,8 @@ func (p *Plugin) outcome(outctx ocr3types.OutcomeContext, query types.Query, aos
 				if p.Config.VerboseLogging {
 					p.Logger.Warnw("Aggregation failed", "aggregator", agg, "channelID", cid, "f", p.F, "streamID", sid, "observations", streamObservations[sid], "stage", "Outcome", "seqNr", outctx.SeqNr, "err", err)
 				}
-				// FIXME: Is this a complete failure?
-				// MERC-3524
+				// Ignore stream that cannot be aggregated; this stream
```

Comment:
I'd also wonder whether that should be a "warn". Tbh this indicates a wrong definition, probably "error" worthy.

Reply:
It does warn if you enabled VerboseLogging, see like 236.

The reason it doesn't always warn or error is because it could be quite spammy e.g. if you had some data source unavailable and consistently didn't have enough valid observations to aggregate. It could also be the fault of other nodes so warning on this node may not be particularly useful.